### ISSUE TITLE
Unable to use version 0.3.0 - not published correctly?

### ISSUE THREAD

**Issue by balonik** [disc/terraform-provider-pritunl#66]
While doing `terraform init` I get following error:

```
| Error: Failed to install provider
│ 
│ Error while installing disc/pritunl v0.3.0: checksum list has unexpected SHA-256 hash 420cff96fe96c94f6d93e0cfea1192ef66f300d2ed88711d6734b78fcacca030 (expected
│ 49f7d9b8ed6142d665820bc9c72b37b57243df7fa4172d9bf66a0702ec2aa048)
```

**Comment by disc** [comment]
Hey @balonik, it looks weird, something wrong happened yesterday after a release 0.2.1 and 0.3.0 version.
I've just released v0.3.1 that targets to the latest commit, can you try a new version?

**Comment by balonik** [comment]
Hi @disc, 0.3.1 is pulled OK during init, thanks!
